tests/python_package_test/test_engine.py

```diff
@@ -15,7 +15,13 @@
 import psutil
 import pytest
 from scipy.sparse import csr_matrix, isspmatrix_csc, isspmatrix_csr
-from sklearn.datasets import load_svmlight_file, make_blobs, make_multilabel_classification
+from sklearn.datasets import (
+    load_svmlight_file,
+    make_blobs,
+    make_classification,
+    make_multilabel_classification,
+    make_regression,
```

Comment:
```suggestion
```

Instead of introducing an import of `sklearn.datasets.make_regression`, please use the wrapper we have around it:

https://github.com/microsoft/LightGBM/blob/480600b3afaf2a0a6f32cf417edf9567f625b2c3/tests/python_package_test/utils.py#L118-L122

We like to do that because it makes the tests run faster... every invocation after the first one gets a cached dataset in memory instead of re-generating a new one.

Reply:
Thank you for kind review 👍 , I removed the `make_regression`.